tests/system/get-command.test.ts

```diff
@@ -3,26 +3,112 @@ import { test } from 'tap';
 import { getCommand } from '../../lib';
 import * as os from 'os';
 
-test("should return 'mvn' when no mvnw present in path", async (t) => {
-  const cmd = getCommand(
-    '.',
-    path.join(__dirname, '..', 'fixtures/path with spaces', 'pom.xml'),
-  );
+const SCENARIOS = [
+  {
+    name: "should return 'mvn' when no mvnw present in path",
+    root: '.',
+    targetFilePath: path.join(
+      __dirname,
+      '..',
+      'fixtures/path with spaces',
```

Comment:
will this work on Windows tests? Maybe separate into 2 segments for `path.join`

Reply:
Same goes for other scenarios

Reply:
it does. These are all strings that I copied from the way tests were prev written